iaso/api/workflows/import_export.py

```diff
@@ -76,9 +76,15 @@ def import_workflow_real(workflow_data: typing.Dict, account: Account) -> Workfl
     except Workflow.DoesNotExist:
         wf = Workflow.objects.create(uuid=workflow_data["uuid"], entity_type=entity_type)
 
+    # Set all related WorkflowVersion objects' status to UNPUBLISHED if they are currently PUBLISHED
+    published_versions = wf.workflow_versions.filter(status=WorkflowVersionsStatus.PUBLISHED.value)
+    for version in published_versions:
+        version.status = WorkflowVersionsStatus.UNPUBLISHED.value
```

Comment:
This assumes that the import always include a published workflow that will replace the previously published ones, I think. Is it always true?